hudi-common/src/main/java/org/apache/hudi/common/util/NumericUtils.java

```diff
@@ -31,4 +41,28 @@ public static String humanReadableByteCount(double bytes) {
     String pre = "KMGTPE".charAt(exp - 1) + "";
     return String.format("%.1f %sB", bytes / Math.pow(1024, exp), pre);
   }
+
+  public static long getMessageDigestHash(final String algorithmName, final String string) {
+    MessageDigest md = null;
+    try {
+      md = MessageDigest.getInstance(algorithmName);
+    } catch (NoSuchAlgorithmException e) {
+      LOGGER.error("Invalid Algorithm Specified: {}", algorithmName);
```

Comment:
How about removing the LOGGER and throwing exception here? though Objects.requireNonNull(md) would also throw exception if md is null.

Reply:
ok will do